sdc/tests/test_series.py

```diff
@@ -1284,6 +1258,46 @@ def test_impl(A):
         S = pd.Series([2, 4, 6, 6, 3], [1, 3, 5, 13, 22])
         pd.testing.assert_series_equal(hpat_func(S), test_impl(S))
 
+    @skip_sdc_jit('Not impl in old style')
+    def test_series_slice_loc_start(self):
+        def test_impl(A, n):
+            return A.loc[n:]
+        hpat_func = self.jit(test_impl)
+
+        all_data = [[1, 3, 5, 13, 22], [1, 3, 3, 13, 22]]
+        key = [1, 3, 18]
+        for index in all_data:
+            for n in key:
+                S = pd.Series([2, 4, 6, 6, 3], index)
+                pd.testing.assert_series_equal(hpat_func(S, n), test_impl(S, n))
+
+    @skip_sdc_jit('Not impl in old style')
+    def test_series_slice_loc_stop(self):
+        def test_impl(A, n):
+            return A.loc[:n]
+        hpat_func = self.jit(test_impl)
+
+        all_data = [[1, 3, 5, 13, 22], [1, 3, 3, 13, 22]]
+        key = [1, 3, 18]
+        for index in all_data:
+            for n in key:
+                S = pd.Series([2, 4, 6, 6, 3], index)
+                pd.testing.assert_series_equal(hpat_func(S, n), test_impl(S, n))
+
+    @skip_sdc_jit('Not impl in old style')
```

Comment:
Please add tests also for decreasing index

Reply:
I added tests for decreasing index and they failed. Need to rewrite the algorithm